modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -59,4 +59,58 @@ test.describe('Manage object fields through Model Builder', () => {
 			ListTypeDefinition.id
 		);
 	});
+
+	test('can delete object field', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		await page.goto('/');
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default');
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.openObjectFolder('default');
+
+		await viewObjectDefinitionsPage.viewInModelBuilder();
+
+		await modelBuilderPage.leftSidebarItems
+			.filter({hasText: objectDefinition.name})
+			.click();
+
+		await modelBuilderPage.createObjectField({
+			mandatory: false,
+			objectDefinitionName: objectDefinition.name,
+			objectFieldBusinessType: 'Integer',
+			objectFieldLabel: 'intField',
+		});
+
+		await modelBuilderPage.leftSidebarItems
+			.filter({hasText: objectDefinition.name})
+			.click();
+
+		await modelBuilderPage.objectDefinitionNodes
+			.filter({hasText: objectDefinition.name})
+			.getByText('textField')
```

Comment:
Hi @rafaellajordao, what do you think about delete the same field that was created on top of the test?